README.md

```diff
@@ -0,0 +1,4 @@
+# winter-of-contributing
```

Comment:
You'll have to write the README file properly @shreyarai05 . It should include proper introduction of the topic, the approach that you're following in your code, advantages over other methods and conclusion. Please do the same otherwise this won't be considered as a quality PR.